submissions/2025-07-01-MLPepper-RECAP-Optimized-Fragments-v1.1/ds_info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "dataset_name": "MLPepper RECAP Optimized Fragments v1.1",
+    "dataset_tagline": "Single point property calculations for charge models, expanded to include iodine.",
+    "description": "A single point dataset created by combining the [50k ESP from Simon](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2022-01-16-OpenFF-ESP-Fragment-Conformers-v1.0) and [Br substituted set from Lily](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2023-11-30-OpenFF-multi-Br-ESP-Fragment-Conformers-v1.1-single-point). This was then extended to cover Boron and Silicon by applying the RECAP decomposition scheme in RDKit to the [B & Si pubchem set from SPICE2](https://github.com/openmm/spice-dataset/blob/main/pubchem/pubchem-boron-silicon.hdf5). This dataset now contains additional iodine containing molecules. Each fragment had 5 conformations generated which were optimised locally using an AIMNET2 model trained to `wb97m-d3`.\nThe aim of the dataset is to provide polarised and gas phase electrostatic properties which can be used to generate ML models  for partial charge prediction. Unlike past datasets the wavefunction will not be saved to recompute the ESP instead we recommend building the ESP  from the MBIS atomic multipoles which save substantial amount of space.  An off equilibrium data set will also be generated to enable conformation dependent prediction of charges.",
+    "metadata.submitter": "jaclark5",
+    "metadata.long_description_url": "https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2025-05-29-OpenFF-SMIRNOFF-Sage-2.0.0"
```

Comment:
Could you please just update the description in ds_info.json and in the overall README to include something about combining the previous 2 datasets? This URL is also out of date. Otherwise LGTM, thanks!